sycl/doc/extensions/proposed/sycl_ext_oneapi_annotated_arg.asciidoc

```diff
@@ -157,14 +153,14 @@ only the top-most level `annotated_arg` properties will be applied to the
 kernel arguments.
 
 In the example above, only properties `PropA` and `PropB` will be applied to the
-kernel argument and the other properties on members of `MyType` will not be
+kernel arguments and the other properties on members of `MyType` will not be
 processed.
```

Comment:
This wording implies that it is legal to use `annotated_arg` in the way you decorate `PropB` and `PropC` and that the implementation is required to ignore those properties.  I think we want to say instead that this usage is illegal and allow the implementation to flag this as an error.

Reply:
Perfect! Updated to clarify that this is illegal.